KubeArmor/.goreleaser.yaml

```diff
@@ -7,7 +7,6 @@ builds:
       - linux
     goarch:
       - amd64
```

Comment:
should this be removed too? so goreleaser can build based on machine arch?

Reply:
ig template is not supported here for build configs, keeping it as it is for now.